### PR TITLE
Update readme

### LINKED ISSUE #6: Update Readme Documentation
The Readme has old build processes and vision.

### PR DESCRIPTION
**Pull request by duckbrain**
Updates the documentation in the README to correct the build process and provide more up to date information about the project and help needed.

FIxes #6

### CODE REVIEW

**Comment by truesilver92** [comment]
Should the church link be to scriptures.lds.org (the gosple library)?

File: README.md [hunk 1]
```diff
@@ -1,40 +1,52 @@
 # LDS Scriptures
 
-LDS Scriptures port to native command line in Go.
+![](data/web/static/favicon.ico)
 
-LDS Scriptures is a set of tools for downloading, parsing, and reading the Gospel Library content from [The Church of Jesus Christ of Latter-day Saints](http://lds.org). 
+ldss is a set of tools for downloading, parsing, and reading the Gospel Library content from [The Church of Jesus Christ of Latter-day Saints](http://lds.org).
```
Comment (truesilver92) [inline README.md:5]: Should this be scriptures.lds.org (a direct link to the gospel library)?

**Comment by truesilver92** [comment]
Other than the link (which is a discretionary thing) I think this is ready to merge. I would like to do the merge for practice once you are ready if that is fine.

Any special merge instructions?
Are you happy with the `git merge --no--ff`?

File: README.md (hunk 1 above)
Comment (duckbrain) [reply README.md:5]: The copyright content is also displayed here throughout the site. There are two apps. This is probably the best link for reference to contact or understand origin.

**Comment by duckbrain** [comment]
Please do merge it. I'm fine with a merge commit.

**Pull request merged**